code/_core/obj/item/contract/_contract.dm

```diff
@@ -107,18 +107,22 @@
 
 /obj/item/contract/proc/turn_in(var/mob/living/advanced/player/P,var/params)
 	var/turf/T = get_turf(P)
-	if(burgerbux_reward)
-		P.to_chat(span("notice","You are awarded \the [reward.name] and [burgerbux_reward] burgerbux for completing the contract."))
-		P.adjust_burgerbux(burgerbux_reward)
+	if(src.value_current <= src.value_max)
+		P.to_chat(span("warning","You feel it would be unwise to try to turn in a contract that isn't complete yet!"))
+		return FALSE
```

Comment:
I made a mistake, which is my bad. It should actually be < instead.

Reply:
And I made the mistake of being to tired to logic check.